<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>F_ImprimerEtiquetteJeu</class>
 <widget class="QDialog" name="F_ImprimerEtiquetteJeu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>423</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Impression d'une étiquette d'un jeu</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="Lb_NomJeu">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Nom du jeu</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="Lb_InfoLudo">
         <property name="text">
          <string>InfoLudo</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="Lb_Code">
       <property name="text">
        <string>Code :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Le_CodeJeu">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="Lb_Contenu">
       <property name="text">
        <string>Contenu</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="TxE_Contenu"/>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCommandLinkButton" name="Bt_Imprimer">
       <property name="text">
        <string>Imprimer cette étiquette</string>
       </property>
       <property name="icon">
        <iconset resource="ImagesIcones.qrc">
         <normaloff>:/Valider.png</normaloff>:/Valider.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCommandLinkButton" name="Bt_Quitter">
       <property name="text">
        <string>Retour au jeu</string>
       </property>
       <property name="icon">
        <iconset resource="ImagesIcones.qrc">
         <normaloff>:/Annuler.png</normaloff>:/Annuler.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="ImagesIcones.qrc"/>
 </resources>
 <connections/>
</ui>
